app/models/application_letter.rb

```diff
@@ -46,7 +55,21 @@ def after_deadline?
   # @param none
   # @return [Boolean] true if status changes are allowed
   def status_change_allowed?
```

Comment:
Its a helper for validation functions, and we have (and introduce) a lot of validation tests already. I dont think we should test helpers separatly